controllers/apply/awards-for-all/form.js

```diff
@@ -318,136 +293,170 @@ module.exports = function({ locale, data = {} }) {
+            title = localise({
+                en: 'Type of statutory body',
+                cy: ''
+            });
+        }
+
+        return {
+            title: title,
+            fieldsets: [
+                {
+                    legend: title,
+                    fields: includeIfOrganisationType(
+                        ORGANISATION_TYPES.STATUTORY_BODY,
+                        [fields.organisationSubTypeStatutoryBody]
+                    )
                 }
-            },
-            {
-                title: localise({ en: 'Registration numbers', cy: '' }),
-                fieldsets: [
-                    {
-                        legend: localise({
-                            en: 'Registration numbers',
-                            cy: ''
-                        }),
-                        fields: compact([
-                            includeCompanyNumber() && fields.companyNumber,
-                            includeCharityNumber() && fields.charityNumber,
-                            includeEducationNumber() && fields.educationNumber
-                        ])
-                    }
-                ]
-            },
-            {
-                title: localise({ en: 'Organisation finances', cy: '' }),
-                fieldsets: [
-                    {
-                        legend: localise({
-                            en: 'Organisation finances',
-                            cy: ''
-                        }),
-                        fields: compact([
-                            includeAccountDetails() &&
-                                fields.accountingYearDate,
-                            includeAccountDetails() && fields.totalIncomeYear
-                        ])
-                    }
-                ]
-            }
-        ]
-    };
+            ]
+        };
+    }
 
-    const sectionSeniorContact = {
-        slug: 'senior-contact',
-        title: localise({ en: 'Senior contact', cy: '' }),
-        summary: localise({
-            en: `Please provide details for your senior contact. This person will be legally responsible for the funding. They can't be married, in a long-term relationship, living with, or related to the main contact.`,
-            cy: ``
-        }),
-        steps: [
-            {
-                title: localise({ en: 'Senior contact', cy: '' }),
-                fieldsets: [
-                    {
-                        legend: localise({
-                            en: 'Who is your senior contact?',
-                            cy: ''
-                        }),
-                        introduction: localise({
-                            en: `<p>
+    /**
+     * Registration numbers are conditional based on the organisation type
+     * This step can include a combination of company number,
+     * charity number, and/or department for education number.
+     */
+    function stepRegistrationNumbers() {
+        const includeCompanyNumber = [
```

Comment:
Using functions for the steps allows us to move this logic closer to where it's used rather than having `includeCompanyNumber()` functions at the top of the file a mile away from where it's called.